package-lock.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@openinf/util-errors",
   "version": "2.0.7",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

```suggestion
  "lockfileVersion": 3,
```

👎 this causes the Snyk check to fail -- i cannot merge this